src/model/paste/__tests__/DraftPasteProcessor-test.js

```diff
@@ -293,7 +293,7 @@ describe('DraftPasteProcessor', function() {
     expect(output[0].getText()).toBe('This is a link, yep.');
     var entityId = output[0].getCharacterList().get(12).getEntity();
     var entity = DraftEntity.get(entityId);
-    expect(entity.getData().url).toBe('http://www.facebook.com/');
+    expect(entity.getData().url).toBe('http://www.facebook.com');
```

Comment:
The link was `'This is a <a href="http://www.facebook.com">link</a>, yep.`

not sure how did we get to the backslash on the end.